src/Microsoft.DotNet.Darc/src/Darc/Operations/VirtualMonoRepo/ScanOperation.cs

```diff
@@ -13,16 +13,30 @@ namespace Microsoft.DotNet.Darc.Operations.VirtualMonoRepo;
 
 internal class ScanOperation : Operation
 {
+    private readonly bool _scanCloaked;
+    private readonly bool _scanBinary;
+
     public ScanOperation(ScanCommandLineOptions options) : base(options, options.RegisterServices())
     {
+        _scanBinary = options.Binary;
+        _scanCloaked = options.Cloaked;
     }
 
     public override async Task<int> ExecuteAsync()
     {
         var vmrScanner = Provider.GetRequiredService<IVmrScanner>();
         using var listener = CancellationKeyListener.ListenForCancellation(Logger);
 
-        await vmrScanner.ListCloakedFiles(listener.Token);
+        if (_scanCloaked)
```

Comment:
I am also thinking of having 2 commands instead of merging these. I can imagine, in the pipeline, we might want to have 2 different build steps so that it's immediately clear if we have cloacked files and binaries. So 0-2 orange steps ideally.

`vmr scan-binaries`
`vmr scan-cloaked-files`

Reply:
I split them